lib/generators/rolemodel/modals/README.md

```diff
@@ -2,31 +2,33 @@
 
 ## What you get
 
-* Custom confirms
-* Modal and panel styling
-* Modal and panel view and link helper
-* Modal and panel javascript
+* Custom confirm dialogs via `@rolemodel/turbo-confirm` integration
+* Modal and panel styling ..via `@rolemodel/optics` integration?
+* Modal and panel layouts w/ link helpers (or you could just, e.g. `link_to ... data: {turbo_frame: 'modal'}`)
```

Comment:
It's my opinion that the link helpers should be removed entirely.

* I believe they should be removed because they obscure how modals and panels work, making them seem more magical than they really are.
* Links often require application/context specific customization.  Which is likely to lead to bloated link helpers as they are adapted to accommodate application specific use-cases.
* I think it's better to use `link_to` directly and ask developers to add `data-turbo-frame='modal'` attributes explicitly.

Does anyone else have an opinion on this?